src/librbd/api/Trash.cc

```diff
@@ -84,6 +84,10 @@ int enable_mirroring(IoCtx &io_ctx, const std::string &image_id) {
     return r;
   }
 
+  if ((features & RBD_FEATURE_JOURNALING) == 0) {
```

Comment:
Hmm ... so we don't have state available to restore the original mirror mode if you move it to the trash?

Reply:
We enable mirroring here only if the pool in mirror mode "pool", which we decided should only be used for journal mirroring.

Reply:
Ack -- thx